vue.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  publicPath: process.env.PUBLIC_URL || '',
+  publicPath: '',
```

Comment:
Just checking if my understanding is correct but we don't need `process.env.PUBLIC_URL` anymore because we're not relying on env variables to do github page deployments for PRs right?

Reply:
Yes that's right.